config/redirects.txt

```diff
@@ -4,128 +4,242 @@
 
 # Legacy URLs
 /pages/ /
+/pages /
```

Comment:
IMHO I don't think we should commit these in in. It makes a pattern where folks unassuming will add redirects with or without slash then duplicate them without knowing why. On having these done build time we can create a predictable file that we know is taking care of all the cases. It should also be clear here why this is even happening because this is not an ideal behavior by Cloudflare Pages. This should have been handled quite easily if Cloudflare provided support for regex which they told me could be coming soon™️

Reply:
As discussed previously, I don't like the idea of redirects being generated behind the scenes that we may not understand, especially if the regex is wrong or breaks. I would much rather have the redirects be in source control and put the burden on the developer to add the required variants.

I want to avoid a case where redirects start being weird in production and no one knows why because a regular expression is doing the heavy lifting.

Reply:
Another way to look at it is, assume the regex is bad, or goes bad, I would rather block a developer trying to merge a PR, then break something in production.

Reply:
Okay that last part I agree, but then I recommend we add some instruction into how exactly folks will need to add a new redirect. Should they add only the trailing slash redirect and generate the one without trailing slash?

Reply:
I don't think developers should have to generate anything (ideally). If adding a redirect they should add both variations or they will get met with an error. I think re-generating the file with the provided commands should be a last resort.

Reply:
> Should we document the requirement at the top of the redirects file?

Yeah, that's my point. I think a couple of sentences might help in the redirects file and error.

Reply:
You can make these changes and should be good to go. Do merge it so we can see the changes on Monday.

Reply:
Added a couple sentences to the top of the redirects file. Let me know if it makes sense to you.